discord_slash/model.py

```diff
@@ -185,16 +185,16 @@ class BasecommandObject(CommandObject):
     .. warning::
         Do not manually init this model.
 
-    :ivar has_subcommand: Indicates whether this base command has subcommands.
-    :ivar default_permissions: Indicates whether users should have permissions to run this command by default.
+    :ivar has_subcommands: Indicates whether this base command has subcommands.
```

Comment:
Is this another breaking change?

Reply:
It was `has_subcommands` in the current code, so I think the error without an `s` was typo caused by one of by commits.
https://github.com/eunwoo1104/discord-py-slash-command/blob/d5cba045a60aa5f9e3e16d16e7a4520fbdd31e3c/discord_slash/model.py#L126